lang/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ set(sclang_sources
 	${CMAKE_SOURCE_DIR}/common/SC_StringBuffer.cpp
 	${CMAKE_SOURCE_DIR}/common/SC_StringParser.cpp
 	${CMAKE_SOURCE_DIR}/common/SC_TextUtils.cpp
+	${CMAKE_SOURCE_DIR}/common/localsocket_utils.cpp
```

Comment:
Do we actually need this line? 'lang' seems not to have any changes: all the changes are in 'sc-ide'.